tests/integration/acp/register_and_delete_test.go

```diff
@@ -215,7 +216,7 @@ func TestACP_CreateWithIdentityAndDeleteWithIdentity_CanDelete(t *testing.T) {
 		Actions: []any{
 			testUtils.AddPolicy{
 
-				Creator: OwnerIdentity,
+				Creator: OwnerSignature,
```

Comment:
todo: similar to previous todo update all occurrences of signature to identity.

Reply:
Currently, we represent the `identity` internally as a `string`. If that is what it will always be, then `identity` everywhere is fine. If we foresee that the type will evolve from a `string` to something more complex (which was previously discussed), then it doesn't make sense to call the Auth `string` that comes in with a request the `identity`. It has to have a different name and `signature` would make sense. Or maybe `token` would be appropriate.

Reply:
I look at Identity as whatever bits and pieces belong to identifying an actor. For now that is just the Public Sourcehub Account Address (string). Soon this might be a combination of Priv/Pub key pair. So Identity IMO encapsulates this well.

Signature doesn't make sense if it can have multiple "signatures" in the identity struct. IIRC on protocol side of things signature means something else which would always confuse inter team communication when I kept referring to the sourcehub address as signature.

Reply:
Agree with shahzad here, "Identity" is already the umbrella term, when we get to adding proper JWS based request tokens, this is still "Identity" just a verifable one.

At the end of the day, for Identity, we only need a unique identifier, and a verified public key (and potentially a private key, but that is only for embedded database flows, which we can ignore for this convo)